ui/packages/consul-ui/app/components/consul/lock-session/list/index.hbs

```diff
@@ -13,6 +18,7 @@
       </dt>
       <dd>{{item.ID}}</dd>
     </dl>
+    {{/if}}
```

Comment:
I know this is from ages ago, but could we move the CopyButton here out of the `dt` and into the `dd`?

Reply:
I'm moving away from using `dl` in the composite-row. It's caused a lot of issues. If I update this is to move everything into a `span`. We can discuss this.

Reply:
Leaving any `dl` changes outside of this PR.